contrib/completion/bash/docker

```diff
@@ -469,6 +480,7 @@ __docker_tasks() {
 }
 
 # __docker_complete_services_and_tasks applies completion of services and task IDs.
+# shellcheck disable=SC2120
```

Comment:
This would be a single line right after the shebang listing all the SC rules that we force to ignore globally for the file.

```
#!/bin/bash
# shellcheck disable=SC1091,SC2120,SC2155
```